--- jszip.js	2020-08-12 20:46:01.992475192 +0300
+++ jszip.js	2020-08-12 20:50:21.141679728 +0300
@@ -8,21 +8,12 @@
 
 JSZip uses the library pako released under the MIT license :
 https://github.com/nodeca/pako/blob/master/LICENSE
+
+Taken from <https://raw.githubusercontent.com/Stuk/jszip/v3.5.0/dist/jszip.min.js>, then formatted, then rewritten as an ES module
 */
 
-!(function (t) {
-  if ('object' == typeof exports && 'undefined' != typeof module) module.exports = t();
-  else if ('function' == typeof define && define.amd) define([], t);
-  else {
-    ('undefined' != typeof window
-      ? window
-      : 'undefined' != typeof global
-      ? global
-      : 'undefined' != typeof self
-      ? self
-      : this
-    ).JSZip = t();
-  }
+export default (function (t) {
+  return t();
 })(function () {
   return (function s(a, o, h) {
     function u(r, t) {
